dashboard/test/ui/features/teacher_tools/course_versions.feature

```diff
@@ -124,10 +124,3 @@ Scenario: Switch versions using dropdown on script overview page
   And element ".assignment-version-title:contains(2018)" is not visible
   And I click selector ".assignment-version-title:contains(2017)" once I see it
   Then I wait until I am on "http://studio.code.org/courses/ui-test-versioned-script-2017/units/1"
-
-@as_student
-@no_mobile
-  #TODO: Figure out this?
-Scenario: Course unit family names redirect to 2019 version
```

Comment:
This tests whether a redirect goes to 2019 curriculum. It doesn't seem that useful to test anymore so I removed.